<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>talkw</class>
 <widget class="QWidget" name="talkw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detoks Conversation box </string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/zas/icons/konversation.png</normaloff>:/icons/zas/icons/konversation.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
border-color: rgb(126, 126, 126);

font: 9pt &quot;Calibri&quot;;
border-style:outset;
border-width: 2px;
border-radius: 7px;

margin: 3px;
background-color: rgb(0, 0, 0);
color: rgb(245, 245, 245);</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(208, 203, 236);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="autoFormatting">
       <set>QTextEdit::AutoAll</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Wy≈õlij</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>Rozmowa z :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
